stl/inc/xstring

```diff
@@ -4335,11 +4444,14 @@ _NODISCARD basic_string<_Elem, _Traits, _Alloc> operator+(
 template <class _Elem, class _Traits, class _Alloc>
 _NODISCARD basic_string<_Elem, _Traits, _Alloc> operator+(
     basic_string<_Elem, _Traits, _Alloc>&& _Left, basic_string<_Elem, _Traits, _Alloc>&& _Right) {
-    if (_Right.size() <= _Left.capacity() - _Left.size() || _Right.capacity() - _Right.size() < _Left.size()) {
-        return _STD move(_Left.append(_Right));
-    } else {
-        return _STD move(_Right.insert(0, _Left));
-    }
+#if _CONTAINER_DEBUG_LEVEL == 1

```

Comment:
I don't think that this is a good match for CDL's intent, which is for lightweight, high-value security checks. This seems like IDL=2 debugging.

Reply:
OK @CaseyCarter you're outvoted :P